index.html

```diff
@@ -1528,12 +1536,12 @@ <h3 id="11" class="css-1uppmgb">About Us</h3>
                                                                                     differences across all spectrums,
                                                                                     including but not limited to
                                                                                     ethnicity, gender, age, disability
-                                                                                    and sexuality. Through our focus on
+                                                                                    and sexuality. Through our focus on Racial
+                                                                                    Equity
                                                                                 </span></span><a
                                                                                 rel="noopener noreferrer"
                                                                                 target="_blank"
-                                                                                href="https://bouldercolorado.gov/services/racial-equity">Racial
-                                                                                Equity</a><span><span>,<span>&nbsp;</span></span><span>our</span><span><span>&nbsp;<a
+                                                                                href="https://bouldercolorado.gov/services/racial-equity">(visit our Office of Equity)</a><span><span>,<span>&nbsp;</span></span><span>our</span><span><span>&nbsp;<a
```

Comment:
this link now properly describes what its purpose is outside of its context, similar to the example above.